### PR TITLE
[Fix] Fix LoggerHooks docstring

### PR DESCRIPTION
**Pull request by teamwong111**
## Motivation

Fix LoggerHooks docstring.

## Modification

As above.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects, like MMDet or MMCls.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmcv/runner/hooks/logger/mlflow.py [hunk 1]
```diff
@@ -6,6 +6,36 @@
...
+            If not None, set tags for the current run.
+        log_model (bool, optional): Whether to log an MLflow artifact.
+            Default True.
+            If True, log runner.model as an MLflow artifact
+            for the current run.
+        interval (int): Logging interval (every k iterations).
+            Default: 10.
+        ignore_last (bool): Ignore the log of last iterations in each epoch
+            if less than `interval`.
+            Default: True.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/mlflow.py:30]:
```suggestion
        ignore_last (bool): Ignore the log of last iterations in each epoch
            if less than `interval`. Default: True.
```

File: mmcv/runner/hooks/logger/mlflow.py [hunk 2]
```diff
@@ -6,6 +6,36 @@
...
+
+    Args:
+        exp_name (str, optional): Name of the experiment to be used.
+            Default None.
+            If not None, set the active experiment.
+            If experiment does not exist, an experiment with provided name
+            will be created.
+        tags (dict of str: str, optional): Tags for the current run.
+            Default None.
+            If not None, set tags for the current run.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/mlflow.py:21]:
```suggestion
        tags (dict of str: str, optional): Tags for the current run.
            Default None. If not None, set tags for the current run.
```

File: mmcv/runner/hooks/logger/mlflow.py [hunk 3]
```diff
@@ -6,6 +6,36 @@
...
+        log_model (bool, optional): Whether to log an MLflow artifact.
+            Default True.
+            If True, log runner.model as an MLflow artifact
+            for the current run.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/mlflow.py:25]:
```suggestion
            Default True. If True, log runner.model as an MLflow artifact
            for the current run.
```

File: mmcv/runner/hooks/logger/mlflow.py [hunk 4]
```diff
@@ -6,6 +6,36 @@
...
+    """Class to log metrics and (optionally) a trained model to MLflow.
+
+    It requires `MLflow`_ to be installed.
+
+    Args:
+        exp_name (str, optional): Name of the experiment to be used.
+            Default None.
+            If not None, set the active experiment.
+            If experiment does not exist, an experiment with provided name
+            will be created.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/mlflow.py:18]:
```suggestion
            Default None. If not None, set the active experiment.
            If the experiment does not exist, an experiment with the provided name
            will be created.
```

File: mmcv/runner/hooks/logger/neptune.py [hunk 5]
```diff
@@ -28,13 +28,20 @@ class NeptuneLoggerHook(LoggerHook):
             Check https://docs.neptune.ai/api-reference/neptune#init for more
             init arguments.
         interval (int): Logging interval (every k iterations).
+            Default: 10.
         ignore_last (bool): Ignore the log of last iterations in each epoch
-            if less than `interval`.
-        reset_flag (bool): Whether to clear the output buffer after logging
+            if less than ``interval``.
+            Default: True.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/neptune.py:34]:
```suggestion
            if less than ``interval``. Default: True.
```

File: mmcv/runner/hooks/logger/tensorboard.py [hunk 6]
```diff
@@ -9,6 +9,23 @@
...
+    Args:
+        log_dir (string): Save directory location.
+            Default: None.
+            If default values are used, directory location is
+            ``runner.work_dir``/tf_logs.
+        interval (int): Logging interval (every k iterations).
+            Default: True.
+        ignore_last (bool): Ignore the log of last iterations in each epoch
+            if less than `interval`.
+            Default: True.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/tensorboard.py:23]:
```suggestion
            if less than `interval`. Default: True.
```

File: mmcv/runner/hooks/logger/tensorboard.py [hunk 7]
```diff
@@ -9,6 +9,23 @@
 
 @HOOKS.register_module()
 class TensorboardLoggerHook(LoggerHook):
+    """Class to log metrics to Tensorboard.
+
+    Args:
+        log_dir (string): Save directory location.
+            Default: None.
+            If default values are used, directory location is
+            ``runner.work_dir``/tf_logs.
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/tensorboard.py:18]:
```suggestion
            Default: None. If default values are used, directory location is
            ``runner.work_dir``/tf_logs.
```

File: mmcv/runner/hooks/logger/pavi.py [hunk 8]
```diff
@@ -15,6 +15,10 @@
 
 @HOOKS.register_module()
 class PaviLoggerHook(LoggerHook):
+    """Class to visual model, log metrics, etc to Pavi.
+
+    Used within OpenMMLab organization only.
+    """
```
Comment (zhouzaida) [inline mmcv/runner/hooks/logger/pavi.py:21]:
```suggestion
    """Class to visual model, log metrics (for internal use).
    """
```

Comment (teamwong111) [reply mmcv/runner/hooks/logger/pavi.py]: Done

**Review by zhouzaida — APPROVED** [review]

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**